sources/@roots/bud-extensions/src/Extensions/extensions.service.ts

```diff
@@ -74,16 +64,10 @@ export class Extensions extends Service implements Base {
       return
     }
 
-    if (this.app.project.peers.hasMissingDependencies) {
```

Comment:
this is worthless now and is the main culprit of resolve issues, i think. i thought i had removed it.